bigtop-packages/src/common/alluxio/install_alluxio.sh

```diff
@@ -131,7 +134,7 @@ cp -a integration/* $PREFIX/$LIB_DIR/integration
 cp integration/fuse/target/alluxio-integration-fuse-*-jar-with-dependencies.jar $PREFIX/$LIB_DIR/integration/fuse
 
 # replace the original libjnifuse*.so file with the manually compiled in openEuler
```

Comment:
The comment should be updated too.

Reply:
@iwasakims Thanks, the comments has updated. https://github.com/apache/bigtop/commit/9f0695218f5a73315101dfac29beef07a80526f5